src/style.css

```diff
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
no need to repeat all styles. just set common styles for `stars__star` and change background using `nth-child` for bem modifiers.